packages/gitbook-docs/src/markdoc/attributes.ts

```diff
@@ -26,6 +26,11 @@ const convertLink = (link: string): string =>
 // eslint-disable-next-line functional/no-classes
 export class LinkAttr {
   readonly transform = (value: string | null, { variables }: Config) => {
+    if (value && value.includes('mailto:')) {
```

Comment:
I think the `startsWith` suite more the use-case (here the value should be `mailto:...`). Moreover some lines below there is already an if, I think we should extend it adding an `else if`

Reply:
ok for the else if, but i'd leave it to be `includes` because i think itìs more future proof given that the links are not hardcoded but they can be configured externally.